src/main/java/org/jitsi/videobridge/VideoChannel.java

```diff
@@ -278,6 +278,22 @@ private static synchronized RecurringRunnableExecutor getRecurringExecutor()
         getRecurringExecutor().registerRecurringRunnable(bandwidthProbing);
     }
 
+    /**
+     * {@inheritDoc}
+     */
+    @Override
+    protected void maybeStartStream()
+        throws IOException
+    {
+        boolean previouslyStarted = getStream().isStarted();
```

Comment:
Could you please add some comments here that explain why this is necessary?

Reply:
Yep, done